xml/System.Runtime.CompilerServices/RuntimeHelpers.xml

```diff
@@ -210,7 +210,7 @@
         <param name="o2">The second object to compare.</param>
         <summary>Determines whether the specified <see cref="T:System.Object" /> instances are considered equal.</summary>
         <returns>
-          <see langword="true" /> if the <paramref name="o1" /> parameter is the same instance as the <paramref name="o2" /> parameter, or if both are <see langword="null" />, or if <c>o1.Equals(o2)</c> returns <see langword="true" />; otherwise, <see langword="false" />.</returns>
+          <see langword="true" /> if the <paramref name="o1" /> parameter is the same instance as the <paramref name="o2" /> parameter, or if both are <see langword="null" />, or if both are the same value type and the values of all fields from both are equal; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if the <paramref name="o1" /> parameter is the same instance as the <paramref name="o2" /> parameter, or if both are <see langword="null" />, or if both are the same value type and the values of the underlying memory from both are equal; otherwise, <see langword="false" />.</returns>
```